README.md

```diff
@@ -10,7 +10,9 @@ The uORB message definitions, which represent the counter-part of the ROS2 messa
 
 ## How are these messsage definitions generated?
 
-The PX4 Firmware Jenkins-based CI/CD deploys new messages or modifications to the current definitions with each build of the master branch of the PX4 Firmware. A future improvement will also include tagging and releasing this repository according to the tags/releases of the PX4 Firmware, for a more tide coupling between the message definitions.
+When uORB message definitions in the [PX4 firmware](https://github.com/PX4/Firmware) repository changes, a Jenkins-based CI/CD automatically generates and pushes updated ROS message definitions to this repository. The ROS message definitions are generated by a script [`uorb_to_ros_msgs.py`](https://github.com/PX4/Firmware/blob/master/msg/tools/uorb_to_ros_msgs.py). You can also use this script to generate your own ROS message definitions, for instance if you have a custom PX4 firmware fork with modified uORB definitions.
+
+A future improvement will also include tagging and releasing this repository according to the tags/releases of the PX4 Firmware, for a more tidy coupling between the message definitions.
```

Comment:
We should remove this from the README and put it under an issue instead. Can you please remove it? Thanks!

Reply:
Absolutely!